src/scripts/main.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
-const people = require('./lib/people');
+const people = require('./lib/people.json');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(human => {
+  const { sex, born, died } = human;
+  const gender = (personSex) => personSex === 'm' ? 'Male' : 'Famele';
+  const nameHuman = human.name;
```

Comment:
an extra variable. You may destruct from an object human as you did with other values
```suggestion
```

Reply:
Not fixed

Reply:
still not fixed

Reply:
still not fixed

Reply:
still not fixed